library/include/rocwmma/internal/types_ext.hpp

```diff
@@ -42,6 +42,38 @@ namespace rocwmma
 {
 
 #if !defined(__HIPCC_RTC__)
+
+    ///////////////////////////////////////////////////////////
+    ///////////  rocwmma::hfloat16_t host conversions  //////////
+    ///////////////////////////////////////////////////////////
+    template<typename Incoming, typename Outgoing , typename std::enable_if_t<!std::is_same<Incoming, hfloat16_t>::value && !std::is_same<Outgoing, hfloat16_t>::value, int> = 0>
+    __host__ __device__ inline Outgoing convert(const Incoming& value)
```

Comment:
If these are host only functions, then we don't need the __device__ attributes. Please use ROCWMMA_HOST instead